README.md

```diff
@@ -1,45 +1,53 @@
-* [atala-prism-wallet-sdk-ts](https://github.com/input-output-hk/atala-prism-wallet-sdk-ts) - Repo that implements Atala PRISM for Browser and Node.js platforms in Typescript.
-* [atala-prism-building-blocks](https://github.com/hyperledger-labs/open-enterprise-agent) - Repo that contains the platform Building Blocks.
+* [atala-prism-wallet-sdk-swift](https://github.com/input-output-hk/atala-prism-wallet-sdk-swift) - Repo that implements
+  Identus for Apple platforms in Swift.
+* [atala-prism-wallet-sdk-ts](https://github.com/input-output-hk/atala-prism-wallet-sdk-ts) - Repo that implements
+  Identus for Browser and Node.js platforms in Typescript.
+* [atala-prism-building-blocks](https://github.com/hyperledger-labs/open-enterprise-agent) - Repo that contains the
+  platform Building Blocks.
 * [atala-prism-mediator](https://github.com/input-output-hk/atala-prism-mediator) - Repo for DIDComm V2 Mediator
 
 ### Modules / APIs
 
-Atala PRISM Kotlin multiplatform SDK provides the following building blocks to create, manage and resolve decentralized identifiers, issue, manage and verify verifiable credentials, establish and manage trusted, peer-to-peer connections and interactions between DIDs, and store, manage, and recover verifiable data linked to DIDs.
+Identus Kotlin multiplatform SDK provides the following building blocks to create, manage and resolve decentralized
+identifiers, issue, manage and verify verifiable credentials, establish and manage trusted, peer-to-peer connections and
+interactions between DIDs, and store, manage, and recover verifiable data linked to DIDs.
 
 * __Apollo__: Building block that provides a suite of cryptographic operations.
 * __Castor__: Building block that provides a suite of DID operations in a user-controlled manner.
 * __Pollux__: Building block that provides a suite of credential operations in a privacy-preserving manner.
-* __Mercury__: Building block that provides a set of secure, standards-based communications protocols in a transport-agnostic and interoperable manner.
+* __Mercury__: Building block that provides a set of secure, standards-based communications protocols in a
+  transport-agnostic and interoperable manner.
 * __Pluto__: Building block that provides an interface for storage operations in a portable, storage-agnostic manner.
-* __PrismAgent__: PrismAgent using all the building blocks provides an agent that can provide a set of high level DID functionalities.
+* __EdgeAgent__: EdgeAgent using all the building blocks provides an agent that can provide a set of high level DID
+  functionalities.
 
 ## Getting started
 
 ### Setup
 
-To get started with the Atala PRISM kotlin multiplatform SDK, you can set up the SDK and start a new project or integrate it into an existing project. Before you start, make sure you have the following installed on your development machine:
+To get started with the Identus kotlin multiplatform SDK, you can set up the SDK and start a new project or integrate it
```

Comment:
`Identus KMM SDK` should be `Edge Agent SDK KMM` according to the [replacement table](https://docs.google.com/spreadsheets/d/1lR3abdg6qExXasUVqBlFouNu9FCB0I1ynMMZodQ6y8U/edit#gid=0)